Contributing.md

```diff
@@ -0,0 +1,141 @@
+
+- If there is no activity on an issue for 5 or more days, and you want to take this up, just ping the assignee in the comments if (s)he is still working. And if there's no response from their end in 24 hrs, we'll assign you to the issue.
+
+### Important
+An important part of working on issues is documenting your work in such a way that it is easy for others to read and understand.
+
+- Commit Guidelines: Before starting your first commit, check out this link: http://coala.io/commit.
+- For more information about how to style Python code according to the PEP8 code style convention, please check the following link: PEP8 Style Guide for Python code: https://www.python.org/dev/peps/pep-0008/
+
+If your commit message doesn't look good, and you wanna make changes to it, read[this](https://github.com/wtfpython-web/wtfpython-web/pull/15#issuecomment-399772374) comment.
+
+# Step 3. Creating a Fork and Testing Your Changes
+
+This tutorial assumes you are working on your own fork. To fork the repository, go to the official repository of wtfpython-web/wtfpython-web and click on the "Fork" button from the website interface.
+
+To add your fork locally, simply run:
+
+$ git remote add myfork fork_link
+
+where myfork is the name of your fork, and fork_link is a link to your fork repository.
+
+### Important
+It is important that you <b>DO NOT</b> make your changes on the master branch of your forked repository to avoid the following cases:
+
+- If you make a rebase to synchronize your repository to the original, every commit that is pushed to the remote master will be pulled in your master branch. Then if you make a pull request to commit your changes to the remote, the commits that got synced from the rebase will be recommitted along with your work in the pull request.
+
+- You cannot have two pull requests using the same branch name. Therefore, if your fork’s master has been used in a pull request and you decide to work on a different issue you will have to branch eventually. Differently every new commit that you make on your master branch will get attached to the initial pull request and that will result in altering the purpose of that request.
+
+- If your fork’s master has been used in a pull request, you have to keep the change in the branch until that get’s merged to the remote master. That will lead to the complications listed above, if you decide to work on a different issue.
+
+In order to avoid the above mentioned cases you can create a new branch where you will work on the issue. To do that run:
+
+$ git checkout -b branchname
+
+# Step 4. Sending Your Changes
+
+### Caution
+
+Before committing your changes, please check that you are indeed in a development branch created in step 3. To check if you are in a branch, type:
+
+$ git branch
+
+Your current branch will have an asterisk (*) next to it. Ensure that there is no asterisk next to the master branch.
```

Comment:
This asterisk should be escaped. It is making all the trailing text in Italics style.